tests/test_obsolete.py

```diff
@@ -297,6 +297,22 @@ async def test(objects):
 
         self.run_with_managers(test)
 
+    def test_get_or_none(self):
+        async def test(objects):
+            text1 = "Test %s" % uuid.uuid4()
+            text2 = "Test %s" % uuid.uuid4()
+
+            obj1 = await objects.create(TestModel, text=text1)
+            obj2 = await objects.get_or_none(TestModel, text=text1)
+            obj3 = await objects.get_or_none(TestModel, text=text2)
+
+            self.assertEqual(obj1, obj2)
+            self.assertIsNotNone(obj1)
+            self.assertIsNotNone(obj2)
+            self.assertIsNone(obj3)
+
+        self.run_with_managers(test)
+
```

Comment:
Thank you for your PR but could you move this test to test_common.py file and rewrite in a pytest way like other ones? The file contains deprecated tests which have to be rewritten  #177

Reply:
I' ve sorted out it's for mysql auth. Resolved

Reply:
done